### ISSUE TITLE
Not receiving email

### ISSUE THREAD

**Issue by dax29** [UksusoFF/webtrees-reminder#51]
Hi
About 2 weeks ago I all of a sudden stopped receiving reminder emails. I have checked with other members they haven't received any either.

So I uninstalled the module and removed the preferences on my site. I then downloaded the latest file (May 2022) and reinstalled. I also had Greg Roach update the Cron file. 

Unfortunately I'm still not receiving any reminder emails?

Any idea what the problem could be
Cheers Frank

Webtrees site: familytree.kuilboer.id.au

**Comment by UksusoFF** [comment]
Try to go link from cron settings in browser. What you see?

**Comment by dax29** [comment]
Hi
Thanks for getting back to me.

I put the following from the cron settings in my browser  "
https://familytree.kuilboer.id.au/reminder-cron/run?key=PkxYxePNajwnBivH"
and got the follwing reply
"{"success":true}"

Cheers
Frank
___________________________________________________________
Frank & Sonia Kuilboer
68 Clark Lane
Broadwater  Queensland  4380 Australia

Mobile: 0439090856 (Frank), 0412214484 (Sonia)
Email: ***@***.*** or ***@***.***
Family History site: http://familytree.kuilboer.id.au/

**Comment by UksusoFF** [comment]
Please check spam folder. And webtrees logs here: https://familytree.kuilboer.id.au/admin/logs
Any errors here?

**Comment by dax29** [comment]
Hi
I've regularly checked my spam folder and nothing in there.

I've attached a copy of the webtrees logs. Hope that helps

Cheers Frank

**Comment by UksusoFF** [comment]
Please attach logs to github or send via email.

**Comment by dax29** [comment]
I attached the log to my last email

**Comment by dax29** [comment]
Here it is again
[webtrees-logs.csv](https://github.com/UksusoFF/webtrees-reminder/files/9824681/webtrees-logs.csv)

**Comment by UksusoFF** [comment]
There is no errors here. Are you using smtp server? 
You received registrations email from webtrees?

**Comment by dax29** [comment]
Well that's interesting, just after I got your email, I started receiving
the reminder email.
Not sure what happened but it seems to be working now
Cheers
Frank

**Comment by dax29** [comment]
I've attached a screenshot shot of the email

**Comment by dax29** [comment]
I don't think it's fixed as I didn't get a reminder today

**Comment by UksusoFF** [comment]
@dax29 please update to latest master. Debug code added there.

**Comment by dax29** [comment]
I updated webtrees to the latest version and  thought it was fixed as I received a Reminder email 2 days in a row but the last 3 days I haven't received any

**Comment by UksusoFF** [comment]
Please update reminder module to master version: https://github.com/UksusoFF/webtrees-reminder/archive/refs/heads/master.zip

**Comment by dax29** [comment]
Thanks for the link, I've updated it now so hopefully that will make it work
Cheers and thanks for your patience

**Comment by UksusoFF** [comment]
There is count 6 on your cron url. So you must get email with 6 records when url triggered.
Are you recieved email when going to url?

**Comment by dax29** [comment]
Sorry not sure what you mean by when going to url?

**Comment by UksusoFF** [comment]
Open link from this comment in browser: https://github.com/UksusoFF/webtrees-reminder/issues/51#issuecomment-1283068896

**Comment by dax29** [comment]
Yes when I put the url in the browser, I get the following result in the
browser {"success":true,"count":6}

I also received the reminder email.

**Comment by UksusoFF** [comment]
Ok, so seems you have troubles with cron. Show your cron config.

**Comment by dax29** [comment]
Is this what you need.
0 9 * * * wget -O - -q "
https://familytree.kuilboer.id.au/reminder-cron/run?key=PkxYxePNajwnBivH"

I have my site hosted with Greg Roach and i got him to set up the Cron
config so would assume he set it up correctly

**Comment by UksusoFF** [comment]
I need output `crontab -e` from self-hosted instance.

**Comment by dax29** [comment]
Sorry how do I get that

**Comment by UksusoFF** [comment]
You need contact with Greg Roach :)
